README.md

```diff
@@ -45,6 +45,11 @@ make
 make test
 ~~~
 
+If `BENCHMARK_collide_spheres_dt` fails by not finding the file `/worlds/collide_spheres.world`, run:
```

Comment:
This is technically correct, but I wonder if we can just add it as a general thing to run... I guess the boxes example didn't need it, if I remember correctly.
I can see adding this either as a general catch-all to run, or as a line in a "troubleshoot" section at the end.
@scpeters has the final word.